src/v3/src/hooks/useInteractionCodeFlow.ts

```diff
@@ -94,6 +101,7 @@ export const useInteractionCodeFlow = (
         status: IdxStatus.SUCCESS,
         ...redirectParams,
       });
+      setCookieUserAuthenticated();
```

Comment:
If you update the location of where the new cookie is set (based on my last comment), then you wont need this here since this hook calls that global success function.

Reply:
addressed: https://github.com/okta/okta-signin-widget/pull/3434/commits/e0ff2c0d00c8b9d7301fde3c0bb75d9b516ccbd9